neuralprophet/plot_forecast.py

```diff
@@ -81,7 +81,7 @@ def plot(
     else:
         fig = ax.get_figure()
     ds = fcst["ds"].dt.to_pydatetime()
-    yhat_col_names = [col_name for col_name in fcst.columns if "yhat" in col_name]
+    yhat_col_names = [col_name for col_name in fcst.columns if "yhat" in col_name and "%" not in col_name]
```

Comment:
will this not affect regular plotting of quantiles?

Reply:
You are right! Pls see PR #764 where I found a better solution